app/views/parents/student-progress/CodeSolutionModal.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <modal
+    title="Code - Solution"
+    @close="$emit('close')"
+  >
+    <div class="code-sol">
+      <code-diff
+        :language="language"
+        :code-right="code"
+        :code-left="solution"
+        />
+    </div>
+  </modal>
+</template>
+
+<script>
+import Modal from '../../../components/common/Modal'
+import CodeDiff from '../../../../ozaria/site/components/teacher-dashboard/common/CodeDiff'
```

Comment:
We should move any Ozaria Components that we use to `app/components` so that we aren't requiring into the Ozaria side of things from the CodeCombat side.